documentation/3_Layers_of_the_Reference_Architecture_Model/3_3_Process_Layer/3_3_6_Policy_Enforcement.md

```diff
@@ -1,4 +1,39 @@
+
+![image](media/uc-example-Components.drawio.png)
+
+_Figure 3.3.6.3: Usage Control Components inside an IDS Connector_
+
+![image](media/uc-example-Sequence.drawio.png)
```

Comment:
Why is there a distinction between an IDS PEP and a PEP, local PIP and IDS local PIP, Log PXP and IDS Log PXP? I do not find the distinction or an explanation in the text....

Reply:
Added